common/utils/searchResults.utils.ts

```diff
@@ -34,8 +34,8 @@ export const sortSearchByTopic = (results: NewWord[]): NewWord[] => {
 const getSearchPositionValue = (search: string, labels: Labels): number => {
   const equal = 0;
   const equalWithin = 1;
-  const endsWith = 2;
-  const startsWith = 3;
+  const startsWith = 2;
+  const endsWith = 3;
   const contains = 5;
```

Comment:
There is one case missing here and that is if the search is null/whitespace 

If you add this one something like` if( (!search.trim()) ){  return results;  }`

Reply:
The search is already trimmed and null-checked before going into this function. So, if a user only types a whitespace (or more) into the search field, this function will not be executed and the results won't update